packages/terra-dev-site/src/pages/page/layouts/CardLayout.module.scss

```
@@ -11,17 +12,6 @@
     position: relative;
   }
 
-  // Firefox bug with overflow content and padding-bottom.
-  // Without this ::after content, the card will not have appropriate
-  // bottom padding when it overflows.
-  // https://bugzilla.mozilla.org/show_bug.cgi?id=1517825
-  .card-layout::before,
-  .card-layout::after {
-    content: '';
-    display: block;
-    height: 15px;
-  }
-
   .card-container:not(:first-child) {
     margin-top: 15px;
```

To correct when there are multiple cards on the page and not having double margins between stacked cards, this now needs to counter and offset each card getting margins:

```suggestion
    margin-top: -15px;
```